docs/guides/docker.md

```diff
@@ -13,9 +13,8 @@ docker run ghcr.io/astral-sh/uv --help
 uv can be installed by copying from the official Docker image:
 
 ```dockerfile
-FROM ghcr.io/astral-sh/uv:latest as uv
 FROM python:3.12-slim-bullseye
-COPY --from=uv /uv /bin/uv
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /bin/uv
```

Comment:
This change might be controversial, as maybe we actually prefer to rely on a named stage?